### ISSUE TITLE
Missing LGBT flag emoji 🏳️‍🌈

### ISSUE THREAD

**Issue by Depaulicious** [Maestroschan/emoji-selector-for-gnome#27]
Hello,
I would like to report that the LGBT rainbow flag (🏳️‍🌈) does not show up in the selector, even though it's present in my emoji font and I can find it using `ibus-uniemoji`.

Have a nice day

**Comment by Depaulicious** [comment]
I have Ubuntu Bionic and full emoji support, works well with Android Oreo Noto Color Emoji font.

**Comment by amivaleo** [comment]
How can you have Bionic? o.O
It's way too early!

**Comment by Maestroschan** [comment]
I never understood how Ubuntu development versions could be used on a daily basis but some people actually use it, so i guess it's not impossible

**Comment by amivaleo** [comment]
I downloaded the iso from here: http://cdimage.ubuntu.com/daily-live/current/HEADER.html
I installed that on gnome-boxes and... No coloured emojis... :(

Depaulicious, please... Share your knowledge...

**Comment by amivaleo** [comment]
Ups.
Forget that...
I only had to install the right font! ^_^

**Comment by Depaulicious** [comment]
@Maestroschan Ubuntu dev branches have always been very stable after the big bugs introduced with Unity were fixed. The GNOME Shell switch one or two releases ago was very smooth I must say. I usually switch to it when they pull in updates with new features for software I use often: bionic worked around (very uglily but it works) Tilix 1.7+ crash on startup, added GTK color emoji, some useful mutter experimental features (fractional DPI scaling, window tiling) and updated some GNOME apps; so I switched to it quite early.

@amivaleo Pull `/system/fonts/NotoColorEmoji.ttf` (or something like that) from your Android device and place it in `~/.fonts`. Then `fc-cache -frv`.
